phpunit.xml

```diff
@@ -1,6 +1,15 @@
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/4.8/phpunit.xsd">
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/5.7/phpunit.xsd"
```

Comment:
`composer.json` has minimum PHPUnit version as 5.7, so the schema change is good.